src/plots/rectanglePlot.ts

```diff
@@ -251,6 +253,50 @@ export module Plots {
       });
     }
 
+    /**
+     * Gets the Accessor for label.
+     */
+    public label(): AccessorScaleBinding<any, string>;
```

Comment:
I realize that this is the case on `symbol()` from `Plots.Scatter` as well, but should we be returning an `AccessorScaleBinding` when there's never a `Scale`? It helps with consistency, but it's kind of odd.